upgrading.md

```diff
@@ -0,0 +1,170 @@
+# Upgrading Philanthropy Data Commons dependencies
```

Comment:
Similarly, upper case filename before the extension here.  (Alternatively, you could move documents to a `docs/` subdir.)